src/common/Navigation/styled.js

```diff
@@ -1,49 +1,160 @@
 import styled from "styled-components";
+import searchIcon from "./searchIcon.svg";
+import { NavLink } from "react-router-dom";
 
-export const NavigationContainer = styled.div`
-    background-color: ${({ theme }) => theme.colors.navigation.background};
+export const NavigationWrapper = styled.div`
+    background: ${({ theme }) => theme.colors.navigation.background};
     color: ${({ theme }) => theme.colors.navigation.elems};
     min-height: 94px;
+`;
 
-    @media(max-width: ${({ theme }) => theme.breakpoints.small}px) {
-        min-height: 142px;
-    }
+export const NavigationContainer = styled.div`
+    max-width: 1368px;
+    margin: 0 auto;
+    min-height: inherit;
 `;
 
-export const Container = styled.div`
+export const DoubleContainer = styled.div`
     display: flex;
-    flex-wrap:wrap;
-    align-items:center;
-    justify-content: center;
-    margin: 0 auto;
-    flex-basis: 50%;
-    padding: 5px;
+    flex-wrap: wrap;
+    min-height: inherit;
```

Comment:
missing:
justify-content: space-between;

@media(max-width: ${({ theme }) => theme.breakpoints.medium}px) {
        justify-content: center;
    }

Reply:
Yes, I am already correcting. Though the real bug was in the left margin: 0 auto in HeadContainer and SearchContainer. Just adding justify-content: space-between didn't change anything on the page. I don't know how I missed it.

Reply:
Yes, I am already correcting. Though the real bug was in the left margin: 0 auto in HeadContainer and SearchContainer. Just adding justify-content: space-between didn't change anything on the page. I don't know how I missed it.